js/test/durfmt/testdurfmt2.js

```diff
@@ -8740,25 +8740,25 @@ module.exports.testdurfmt2 = {
             clockformatted_18.push(textfmt.format({hour: 18,minute: 18,second: 18}).toString());
         }
 
-        test.equal(textformatted_1[0], '‏1 ਸਾਲ, 1 ਮਹੀਨਾ, 1 ਹਫ਼ਤਾ, 1 ਦਿਨ');
-        test.equal(textformatted_1[1], '‏1 ਸਾਲ, 1 ਮਹੀਨਾ, 1 ਹਫ਼ਤਾ, 1 ਦਿਨ');
-        test.equal(textformatted_1[2], '‏1 ਸਾਲ, 1 ਮਹੀਨਾ, 1 ਹਫ਼ਤਾ, 1 ਦਿਨ');
-        test.equal(textformatted_1[3], '‏1 ਸਾਲ, 1 ਮਹੀਨਾ, 1 ਹਫ਼ਤਾ, 1 ਦਿਨ');
+        test.equal(textformatted_1[0], '‏1 y, 1 m, 1 w, 1 d');
```

Comment:
Please make two unit tests: one with script Guru and one with script Arab... no use in throwing out this perfectly good unit test for Guru!

Reply:
We already have test cases for both script testing `Guru` and `Arabic` script. (The test locale is pa-Guru-IN ).